example/helloworld/helloworld.proto

```diff
@@ -38,8 +38,15 @@ package helloworld;
 
 // The greeting service definition.
 service Greeter {
-  // Sends a greeting
+  // A simple RPC
   rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // A response streaming RPC
+  rpc SayHelloGoodbye (HelloRequest) returns (stream HelloReply) {}
```

Comment:
I think these method names are confusing, you simply can't guess cardinality of each method without looking into proto-file.

Reply:
I have renamed method names in the hope they are more understandable.